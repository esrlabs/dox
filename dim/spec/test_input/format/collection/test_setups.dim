document: test_setup

TS_R1:
TS_R2:
TS_R3:
    test_setups:    off_target,off_target
TS_R4:
    test_setups: off_target
TS_R5:
    test_setups:      |
                        off_target,off_target,on_target,on_target
