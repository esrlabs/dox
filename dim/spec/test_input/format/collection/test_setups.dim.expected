document: test_setup

TS_R1:

TS_R2:

TS_R3:
  verification_methods: off_target

TS_R4:
  verification_methods: off_target

TS_R5:
  verification_methods: off_target, on_target
